index.js

```diff
@@ -65,8 +65,7 @@ function addNumericSeparator(num, str) {
     return $replace.call(str, sepRegex, '$&_');
 }
 
-var inspectCustom = require('./util.inspect').custom;
-var inspectSymbol = inspectCustom && isSymbol(inspectCustom) ? inspectCustom : null;
+var inspectSymbol = Symbol.for('nodejs.util.inspect.custom');
```

Comment:
this is very necessary to cover node versions prior to use of `Symbol.for`; additionally, not all browsers support `Symbol.for`.

Also, the `util` package is automatically polyfilled by every non-broken bundler, so this should either Just Work, or, your bundler is broken.